book/src/user/shielded-scan-grpc-server.md

```diff
@@ -0,0 +1,148 @@
+# Zebra Shielded Scanning gRPC Server
+
+## Get Started
+
+### Setup
+
+After setting up [Zebra Shielded Scanning](shielded-scan.md), add a `listen_addr` field to the shielded-scan configuration:
```

Comment:
Will the `shielded-scan.md` link point to the right resource?

If not, let's replace it with:
```suggestion
After setting up [Zebra Shielded Scanning](https://zebra.zfnd.org/user/shielded-scan.html), add a `listen_addr` field to the shielded-scan configuration:
```

Reply:
Thanks for the fix, the previous will work in the github but not in the book,